[WIP] remove abbreviations of german translation

hacki11 opened the pull request:
not complete yet.

- use regexp: [`'’] to find all three apostrophes used
- several words are abbreviated without a dot like '...temp', here we need to do it manually
- some i could not find out:

lines:
616
617 Stabil'? Vielleicht Stabilisierung
619 Verdampfer oder Verdichter?
693 "Anlaufdrehzahl V'lator/Q'pump" (Ventilator/Quellenpumpe?)
873 Vor’schauzeit (keine Ahnung, Vorlaufschauzeit?)

Mit "HK's" sind "alle Heizkreise" gemeint. Vorsicht, auch hier ist ' ` und sonstiges im Einsatz
893 #define STR5013_TEXT "Ladung opt Energie" (opt?)
894 #define STR5016_TEXT "Ladung opt Energie Kontakt" (opt?)
1161 #define STR6137_TEXT "Luftentfeuchter r. F. EIN" (r F EIN?)
1162 #define STR6138_TEXT "Luftentfeuchter r. F. SD"
1251 #define STR6632_TEXT "TA'grenze ext Erzeuger beachten"
1442 #define STR7705_TEXT "Mod'sollwert QX3 Relaistest"
1443 #define STR7705_2_TEXT "Mod'sollwert ZX4 Relaistest" vielleicht Modem?
1905 #define STR9627_TEXT "Gebläse Leistung/Drehzahl Y-Abschn" (Abschnitt? Abschneid..?)

1coderookie commented:
Just really quick: in general, pretty much all names should be available within the detailed description of the specific parameter. If you don't find that specific parameter within that specific manual where you are looking for it in that moment, try another manual or just search for the name of that abbreviation alternatively.  
E.g. from an elco manual: 693 not available in the specific manual I was looking at right now, but the same is mentioned within 3010 = Drehz max V'lator/Q'Pump  = Begrenzt den Regelbereich der **Ventilator- / Quellenpumpen-Drehzahl** gegen oben. -> now we know what V'lator/Q'pump is.

I'll try to complete the list mentioned above tomorrow, then we'll see what else comes up ;)

1coderookie commented:
> 616&617 Stabil'? Vielleicht Stabilisierung  

-> #2838 & #2839: "Stabil'zeit" => "Stabilisierungszeit"

> 619 Verdampfer oder Verdichter?

-> #2841: "Verdichterlaufzeit", nicht 'Verdampfer'

> 693 "Anlaufdrehzahl V'lator/Q'pump" (Ventilator/Quellenpumpe?)

=> korrekt, Ventilator/Quellenpumpe

> 873 Vor’schauzeit (keine Ahnung, Vorlaufschauzeit?)

-> #4743: Ja, ich vermute ebenfalls "Vorlaufschauzeit", da die Funktion beim Schichtschutz zum Einsatz kommt

> Mit "HK's" sind "alle Heizkreise" gemeint. Vorsicht, auch hier ist ' ` und sonstiges im Einsatz

-> imho kann es bei HK's (mit durchgängigem Apostroph) bleiben, ansonsten:
HK / HK's = Heizkreis / Heizkreise
PK = Pumpenkreis
MK = Mischerkreis

> 893 #define STR5013_TEXT "Ladung opt Energie" (opt?)

=> "Ladung optimale Energie" (bezieht sich auf den optimalen Wirkungsgrad)

> 894 #define STR5016_TEXT "Ladung opt Energie Kontakt" (opt?)

=> "Ladung optimale Energie Kontakt" 

> 1161 #define STR6137_TEXT "Luftentfeuchter r. F. EIN" (r F EIN?)

=> "Luftentfeuchter relative Luftfeuchtigkeit EIN" 

> 1162 #define STR6138_TEXT "Luftentfeuchter r. F. SD"

=> "Luftentfeuchetr relative Luftfeuchtigkeit Schaltdifferenz"
-> Anmerkung: Bei etlichen anderen Parametern ist ebenfalls "SD" zu lesen, es ist daher anzunehmen, dass damit ebenfalls die "Schaltdifferenz" gemeint ist

> 1251 #define STR6632_TEXT "TA'grenze ext Erzeuger beachten"

-> Beschreibung, Zitat: 'Zusätzliche über den LPB abgeschlossenen Erzeuger können gemäss eigenen
Parametern aufgrund der Aussentemperatur gesperrt oder freigegeben sein (z.B.
Luft-Wasser-Wärmepumpe).'
=> TA=TemperaturAußen => "Außentemperatur externer Erzeuger beachten"
-> Anmerkung: TA ist auch bei weiteren Parametern zu lesen, auch dort scheint es sich ebenfalls um 'Temperatur Außen' = "Außentemperatur" zu handeln 

> 1442 #define STR7705_TEXT "Mod'sollwert QX3 Relaistest"

=> "Modulationssollwert QX3 Relaistest"

> 1443 #define STR7705_2_TEXT "Mod'sollwert ZX4 Relaistest" vielleicht Modem?

=> "Modulationssollwert ZX4 Relaistest"

> 1905 #define STR9627_TEXT "Gebläse Leistung/Drehzahl Y-Abschn" (Abschnitt? Abschneid..?)

=> Abschnitt => "Gebläse Leistung/Drehzahl Y-Abschnitt"


Korrekturen bzgl der bisherigen 'Übersetzungen' hätte ich auch noch ein paar anzubieten, aber vielleicht geht das dann einfacher, wenn ich nach dem Merge alles nochmal durchgehe und dann einen extra PR starte? @fredlcore: Ich kann dann doch sehen, welche Änderungen vorgenommen wurden, also so wie hier: https://github.com/hacki11/bsb_lan/commit/c1f734d03b1da2ff190290f9e5b3607ad4178057 - oder?

fredlcore commented:
> Korrekturen bzgl der bisherigen 'Übersetzungen' hätte ich auch noch ein paar anzubieten, aber vielleicht geht das dann einfacher, wenn ich nach dem Merge alles nochmal durchgehe und dann einen extra PR starte? @fredlcore: Ich kann dann doch sehen, welche Änderungen vorgenommen wurden, also so wie hier: hacki11@c1f734d - oder?

Ja, Du kannst auf jede Datei im Repo gehen und da auf "History", dann werden immer die Änderungen markiert.

hacki11 commented:
I have updated them, fixed more entries and have 3 left:

#define STR2875_TEXT "Verdichtermod Nach'zeit Tn" (Nachlauf, Nachhol, bin unsicher https://www.mhg-schweiz.ch/de/downloads/aid!7b786239-40bf-4077-9a19-e2c97ebab6c5/id!187/)
#define STR8531_TEXT "Betriebsstunden Kollekt`überhitz" (Kollektorpumpe oder Kollektor?)
#define ENUM8010_41_TEXT "Ladung Elektro, Quell’schutz"

fredlcore commented:
Thanks a lot!

1coderookie commented:
Great, thanks!
Btw:
2875 müsste "Verdichtermodulation Nachstellzeit Tn" sein (da afaik Tn=Nachstellzeit ist)
8531 ist "Betriebsstunden Kollektorüberhitzschutz"
'Quell'schutz' muss ich nochmal nachsehen..
